Docs/API-related/Prompt_Studio_API.md

```
@@ -372,13 +372,52 @@ curl -X POST "http://localhost:8000/api/v1/prompt-studio/optimizations/create" \
       }'
 ```
 
+### Create MCTS Optimization Job (Canary/Flagged)
+```bash
+curl -X POST "http://localhost:8000/api/v1/prompt-studio/optimizations/create" \
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)